src/main/java/org/openrewrite/java/testing/junit4toassertj/ChangeAssertTrue.java

```diff
@@ -0,0 +1,124 @@
+        J.MethodInvocation assertSelect = new J.MethodInvocation(
+                randomId(),
+                null,
+                null,
+                J.Ident.build(randomId(), "assertThat", JavaType.Primitive.Void, EMPTY),
+                new J.MethodInvocation.Arguments(
+                        randomId(),
+                        Collections.singletonList(assertExpression.withPrefix("")),
+                        EMPTY
+                ),
+                null,
+                EMPTY
+        );
+
+        if (originalArgs.size() == 2) {
+            //If the assertTrue is the two-argument variant, we need to maintain the message via a chained method call to "as".
+            //This means the initial method invocation witll be the select into an "as" method: "assertThat(<EXRPESSION>).as(<MESSAGE>)"
+            assertSelect = new J.MethodInvocation(
+                    randomId(),
+                    assertSelect,
+                    null,
+                    J.Ident.build(randomId(), "as", null, EMPTY),
+                    new J.MethodInvocation.Arguments(
+                            randomId(),
+                            Collections.singletonList(original.getArgs().getArgs().get(0)),
+                            EMPTY
+                    ),
+                    null,
+                    EMPTY
+            );
+        }
+
+        //The method that will always be returned is the "isTrue()" method using the assertSelect as the selector.
+        //One argument form : assertThat(<EXPRESSION>).isTrue();
+        //Two argument form : assertThat(<EXPRESSION>).as(<MESSAGE>).isTrue();
+        J.MethodInvocation replacement = new J.MethodInvocation(
+                randomId(),
+                assertSelect,
+                null,
+                J.Ident.build(randomId(), "isTrue", JavaType.Primitive.Boolean, EMPTY),
+                new J.MethodInvocation.Arguments(
+                        randomId(),
+                        Collections.emptyList(),
+                        EMPTY
+                ),
+                null,
+                format("\n")
+        );
+
+        //Make sure there is a static import for "org.assertj.core.api.Assertions.asserThat"
+        maybeAddStaticImport("org.assertj.core.api.Assertions", "assertThat");
+
+        //Format the replacement method invocation in the context of where it is called.
+        andThen(new AutoFormat(replacement));
+        return replacement;
+    }
+
+    /**
+     * This method will add a static import method to the compilation unit.
+     *
+     * @param fullyQualifiedName Fully-qualified name of the class.
+     * @param method The static method to be imported.
+     */
+    public void maybeAddStaticImport(String fullyQualifiedName, String method) {
+        AddImport op = new AddImport();
+        op.setType(fullyQualifiedName);
+        op.setStaticMethod(method);
+        op.setOnlyIfReferenced(false);
+        andThen(op);
```

Comment:
The "maybe" bit of the name is misleading since this will always try to add the import. Perhaps we should add a `maybeAddStaticImport` alongside the existing `maybeAddImport` for scenarios such as this.

Reply:
yep, I wasn't sure if this is was something that you wanted me to generalize (or if I have the correct function signature), so I kept it local for the time being. Would you like me to create a PR to add this to JavaRefactorVisitor?